src/main/java/com/google/roomies/util/StringConverterUtils.java

```diff
@@ -0,0 +1,38 @@
+package com.google.roomies;
+
+import static com.google.roomies.ListingConstants.CURRENCY_CODE;
+import static com.google.roomies.ListingConstants.DATE_FORMAT;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import javax.money.CurrencyUnit;
+import javax.money.Monetary;
+import org.javamoney.moneta.Money;
+
+public final class StringConverterUtils {
+  private StringConverterUtils() {}
+
```

Comment:
Nathan: "utils" here is redundant, prefer brevity. The meaning is equally clear and more concise if the call is StringConverter.stringToDate